components/inputs/input-search.js

```diff
@@ -54,54 +52,23 @@ class InputSearch extends LocalizeCoreElement(RtlMixin(LitElement)) {
 					display: inline-block;
 					width: 100%;
 				}
-				.d2l-input-search-container {
-					position: relative;
-				}
 				:host([hidden]) {
 					display: none;
 				}
-				.d2l-input {
```

Comment:
Nice to get id of some fo this.  🙌

Reply:
Yeah there's a lot of stuff handling the input styling, the slotted clear/search buttons and the "showing focus styles on hover" that is removed.